### PR TITLE
[SDPA-2379] Publication download and print block.

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-2379

### Changed

1.  Publication download and print block.

### Screenshots

Screen Shot 2019-05-09 at 10 31 55-fullpage

### CODE REVIEW

File: packages/Organisms/Publication/PublicationDownloadPrint.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    DocumentLink
+  },
+  props: {
+    links: Array,
+    showPrint: { type: Boolean, default: true },
+    printText: { type: String, default: 'Print this page' }
+  },
+  methods: {
+    printClick () {
+      window.print()
```
Comment (dylankelly) [inline packages/Organisms/Publication/PublicationDownloadPrint.vue]: As discussed, can we emit an event on print? Maybe called publicationPrint? That way we can use it for analytics tracking (I know people can invoke the print window separately too).

File: packages/Organisms/Publication/PublicationDownloadPrint.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<template>
+  <ul class="rpl-publication-download-print">
+    <li v-for="(link, index) in links" :key="index">
+      <rpl-document-link class="rpl-publication-download-print__print-item" v-bind="link" />
```
Comment (dylankelly) [inline packages/Organisms/Publication/PublicationDownloadPrint.vue]: probably need to emit an event here on click too - publicationDownload. With the name of the publication as payload

Comment (alan-cole) [reply packages/Organisms/Publication/PublicationDownloadPrint.vue]: Added an event.

File: packages/Organisms/Publication/PublicationDownloadPrint.vue (hunk 1 above)
Comment (alan-cole) [reply packages/Organisms/Publication/PublicationDownloadPrint.vue]: Added an event.

**Pull request merged**